### PR TITLE
websocket: don't use pooled buffer in mask pool

### PR DESCRIPTION
**Pull request by tsctx**
@mcollina

### CODE REVIEW

File: lib/web/websocket/frame.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ try {
 function generateMask () {
   if (bufIdx === BUFFER_SIZE) {
     bufIdx = 0
-    crypto.randomFillSync((buffer ??= Buffer.allocUnsafe(BUFFER_SIZE)), 0, BUFFER_SIZE)
+    crypto.randomFillSync((buffer ??= Buffer.allocUnsafeSlow(BUFFER_SIZE)), 0, BUFFER_SIZE)
```
Comment (tsctx) [inline lib/web/websocket/frame.js:30]: No problem, but with the current code, I have some concerns.

**Review by ronag — APPROVED** [review]

File: lib/web/websocket/frame.js (hunk 1 above)
Comment (mcollina) [reply lib/web/websocket/frame.js:30]: I don't see them, unless we acess buffer.buffer anywhere

Comment (tsctx) [reply lib/web/websocket/frame.js:30]: If they access it, they can rewrite the mask, right?

Comment (tsctx) [reply lib/web/websocket/frame.js:30]: example:
```js
import { WebSocket } from './index.js'

Buffer.poolSize = 16386 * 4

// any server url
const u = new WebSocket('ws://localhost:3000')

u.onmessage = (data) => {
  // The mask can be rewritten here.
  new Uint8Array(Buffer.allocUnsafe(1).buffer).fill(0)
  u.send('Hi')
}
u.onopen = () => {
  u.send('Hi')
}

```

**Review by KhafraDev — APPROVED** [review]
Is it possible to add a test?

**Comment by tsctx** [comment]
> Is it possible to add a test?

Unfortunately, I can write a test, but I can't catch this completely.

**Comment by tsctx** [comment]
Unfortunately, I can write a test, but this can't catch using pooled buffer completely.

**Comment by KhafraDev** [comment]
better something than nothing

**Comment by tsctx** [comment]
test was placed incorrectly. this can catch it until now!

**Review by ronag — APPROVED** [review]

**Pull request merged**